tools/testing-server/constants.js

```diff
@@ -1,16 +1,39 @@
 const path = require('path')
+const yargs = require('yargs/yargs')
+const { hideBin } = require('yargs/helpers')
+const fs = require('fs')
+const args = yargs(hideBin(process.argv)).argv
 
-const defaultAgentConfig = {
-  licenseKey: 'asdf',
-  applicationID: 42,
-  accountID: 123,
-  agentID: 456,
-  trustKey: 789
+let agentConfig
+if (args.B) {
+  try {
+    agentConfig = JSON.parse(args.B)
+  } catch (err) {
+    try {
+      agentConfig = JSON.parse(fs.readFileSync(args.B, 'utf-8'))
+    } catch (err) {
+      console.error('Failed to parse BAM config:', err)
+    }
+  }
+}
+
+/** fallback if no override was provided or override outright failed to parse correctly */
+if (!agentConfig) {
+  agentConfig = {
```

Comment:
Really to send harvest, only the application ID & licenseKey are needed. To make this .B more useable, you shouldn't need the rest of the properties but simply provide those 2.